app/models/automation_request.rb

```diff
@@ -75,4 +74,20 @@ def self.build_attrs(parameters, user)
     end
   end
   private_class_method :build_attrs
+
+  def self.parse_schedule_options(parameters)
+    {:schedule_type => "schedule"}.tap do |k, _v|
+      if parameters['schedule_time']
+        k[:schedule_time] = Time.zone.parse(parameters['schedule_time'])
+      elsif parameters['schedule_in_days']
+        k[:schedule_in_days] = parameters['schedule_in_days'].to_i.days.from_now
```

Comment:
If we accept a timestamp, why do we need to do the `now + x.days` or `now + x.hours` here?